SocialCareCaseViewerApi/V1/UseCase/MashReferralUseCase.cs

```diff
@@ -43,6 +43,21 @@ public IEnumerable<Boundary.Response.MashReferral> GetMashReferrals(QueryMashRef
                 .Select(x => x.ToResponse());
         }
 
+        public void CreateNewMashReferral(CreateReferralRequest request)
+        {
+            var newContact = new MashReferral
+            {
+                Referrer = request.Referrer,
+                RequestedSupport = request.RequestedSupport,
+                ReferralDocumentURI = request.ReferralUri,
+                Clients = request.Clients,
+                Stage = "CONTACT",
+                CreatedAt = DateTime.Now
```

Comment:
Can we utilise _systemTime.Now instead (I think it makes testing easier as we can mock _systemTime)

Reply:
🙌🏾

Reply:
Done in f2742d9